plugin/pkg/admission/initialresources/admission.go

```diff
@@ -109,14 +110,23 @@ func (ir initialResources) estimateAndFillResourcesIfNotSet(pod *api.Pod) {
 			c.Resources.Requests = api.ResourceList{}
 			req = c.Resources.Requests
 		}
+		annotation := ""
 		if cpu != nil {
 			glog.Infof("CPU estimation for container %v in pod %v/%v is %v", c.Name, pod.ObjectMeta.Namespace, pod.ObjectMeta.Name, cpu.String())
+			annotation = "cpu"
 			req[api.ResourceCPU] = *cpu
 		}
 		if mem != nil {
 			glog.Infof("Memory estimation for container %v in pod  %v/%v is %v", c.Name, pod.ObjectMeta.Namespace, pod.ObjectMeta.Name, mem.String())
+			if annotation != "" {
+				annotation += ", "
+			}
+			annotation += "memory"
 			req[api.ResourceMemory] = *mem
 		}
+		if annotation != "" {
+			pod.ObjectMeta.Annotations[initialResourcesAnnotation] = annotation + " request set by InitialResources plugin"
```

Comment:
If I have more than one container in a pod, and only some of those containers get their resource requirements set, should we state which container had its request set?